static/app/components/events/autofix/preferences/components/SelectedRepoItem.tsx

```diff
@@ -0,0 +1,310 @@
+                    />
+                    {(isEditingBranch ? branchInputValue : settings.branch) && (
+                      <InputGroup.TrailingItems>
+                        <ClearButton
+                          size="xs"
+                          borderless
+                          icon={<IconClose size="xs" />}
+                          onClick={() => {
+                            setBranchInputValue('');
+                            if (!isEditingBranch) {
+                              setIsEditingBranch(true);
+                            }
+                            setIsDirty(true);
+                          }}
+                          aria-label={t('Clear branch and use default')}
+                          title={t('Clear branch and use default')}
+                        />
+                      </InputGroup.TrailingItems>
+                    )}
+                  </InputGroup>
+                </BranchInputContainer>
+              </SettingsGroup>
+
+              <SettingsGroup>
+                <BranchInputLabel>{t('Instructions for Autofix')}</BranchInputLabel>
+                <StyledTextArea
+                  value={instructionsValue}
+                  onChange={handleInstructionsChange}
+                  placeholder={t(
+                    'Any special instructions for Autofix in this repository...'
+                  )}
+                  rows={3}
+                />
+              </SettingsGroup>
+            </RepositorySettingsSection>
+
+            {isDirty && (
+              <FormActions>
+                <Button size="xs" onClick={cancelChanges}>
+                  {t('Cancel')}
+                </Button>
+                <Button size="xs" priority="primary" onClick={saveChanges}>
+                  {t('Save')}
+                </Button>
+              </FormActions>
+            )}
+          </RepoForm>
+        </ExpandedContent>
+      )}
+    </SelectedRepoContainer>
+  );
+}
+
+const BaseRepoContainer = styled('div')`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  border: 1px solid ${p => p.theme.border};
+  border-radius: ${p => p.theme.borderRadius};
+`;
+
+const SelectedRepoContainer = styled(BaseRepoContainer)``;
+
+const SelectedRepoHeader = styled('div')`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: ${space(1.5)};
+  cursor: pointer;
+
+  &:hover {
+    background-color: ${p => p.theme.backgroundSecondary};
+  }
+`;
+
+const RepoNameAndExpandToggle = styled('div')`
+  display: flex;
+  align-items: center;
+`;
+
+const StyledIconExpandToggle = styled(IconExpandToggle)`
+  margin-right: ${space(0.5)};
+`;
+
+const RemoveButton = styled(Button)`
+  color: ${p => p.theme.gray300};
+
+  &:hover {
+    color: ${p => p.theme.red300};
+  }
+`;
+
+const RepoInfoWrapper = styled('div')`
+  display: flex;
+  flex-direction: column;
+  margin-left: ${space(1)};
+`;
+
+const RepoName = styled('div')`
+  font-weight: 600;
```

Comment:
```suggestion
  font-weight: ${p => p.theme.fontWeight.bold};
```
idk if my syntax is correct but you get the idea